cmd/cadvisortosfx/main.go

```diff
@@ -670,7 +670,7 @@ func (swc *scrapWorkCache) waitAndForward() {
 			delete(dims, "kubernetes_container_name")
 		}
 
-		dims["hostHasService"] = dataSourceType
+		dims["sf_hostHasService"] = dataSourceType
```

Comment:
Right now this appears in the UI under a dimension just called "hostHasService", and does not appear under "sf_hostHasService", which is why I asked for the change.